packages/cape/src/methods.ts

```diff
@@ -204,6 +217,28 @@ export abstract class Methods {
     }
   }
 
+  /**
+   * The returned key is stored in the `client` object as a parameter for encrypt. This process happens in the client
+   * locally.
+   *
+   * @example
+   * ```ts
+   * const client = new Cape({});
+   * await key = client.AESkey();
+   * ```
+   */
+  public async AESKey(key?: string) {
+    if (typeof key !== 'undefined') {
+      // We got the key and want to just use it and persist it.
+      this.aesKey = key;
+      return;
+    }
+    // I don't really think we need to generate an AES key
+    // and persisted here. The behavior is that if AES isn't set
+    // explicitly, then we always generate a new AES key for encryption.
+    return;
```

Comment:
```suggestion
    }
```
If the function is void, these returns aren't necessary.